presto-base-jdbc/src/test/java/com/facebook/presto/plugin/jdbc/optimization/TestJdbcComputePushdown.java

```diff
@@ -63,34 +80,114 @@
     private static final Metadata METADATA = MetadataManager.createTestMetadataManager();
     private static final PlanBuilder PLAN_BUILDER = new PlanBuilder(TEST_SESSION, new PlanNodeIdAllocator(), METADATA);
     private static final PlanNodeIdAllocator ID_ALLOCATOR = new PlanNodeIdAllocator();
+    private static final String CATALOG_NAME = "Jdbc";
+    private static final String CONNECTOR_ID = new ConnectorId(CATALOG_NAME).toString();
+
+    private final TestingRowExpressionTranslator sqlToRowExpressionTranslator;
 
     private final JdbcComputePushdown jdbcComputePushdown;
 
     public TestJdbcComputePushdown()
```

Comment:
Can we have more test coverage on 
- `translateConstant`, especially those "?" cases?
- "not" operator